addons/event_sale/views/sale_order_views.xml

```diff
@@ -33,8 +33,17 @@
                     options="{'no_open': True, 'no_create': True}"/>
             </xpath>
             <xpath expr="//field[@name='order_line']//tree//field[@name='product_template_id']" position="after">
-                <field name="event_id" optional="hide" domain="['|', ('company_id', '=', False), ('company_id', '=', parent.company_id)]"/>
-                <field name="event_ticket_id" optional="hide" domain="['|', ('company_id', '=', False), ('company_id', '=', parent.company_id)]"/>
+                <field name="event_id" optional="hide"
+                    domain="[
+                        ('event_ticket_ids.product_id','=', product_id),
+                        ('date_end','&gt;=',time.strftime('%Y-%m-%d 00:00:00')),
```

Comment:
Shouldn't we also allow date_end being False? From what I remember it's not a required field.

Note that I don't really agree with this, date_end should be limiting frontend sales only.
Let's say you are selling VIP tickets for 3 days, then after that period you get a call from someone mentioning that they want a VIP ticket and are ready to pay triple the price, the sale in the backend should be able to sell you such a ticket, even if the sale is technically over.

Do you see what I mean? Maybe to discuss with JIJA.

(Should be removed from the configurator view as well if we go that route).